### PR TITLE
Updates to Migration Guide

### PR DESCRIPTION
**Pull request by blulas**

### CODE REVIEW

**Review by porcelli — CHANGES_REQUESTED** [review]
minor comment, other than that... lgtm

File: migration-guide/now-to-next.adoc [hunk 1]
```diff
@@ -21,7 +21,7 @@ The good news is that this process is relatively painless, as it's driven by Mav
 
 The first step is to generate a new Maven project so that you have a place to migrate your existing business automation assets into the new {PRODUCT_SHORT} architecture.
 There are several ways to create Maven projects, however the easiest is to use a published Maven plugin.
-In order to do that you must first ensure that you have Maven setup properly:
+In order to do that you must first ensure that your have Maven setup properly:
```
[the remark this answers is not in the document]
Comment (LightGuard) [reply migration-guide/now-to-next.adoc]: Agreed, the previous copy was better.

**Review by porcelli — APPROVED** [review]
lgtm

File: migration-guide/now-to-next.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (blulas) [reply migration-guide/now-to-next.adoc]: updated to use "you".

File: migration-guide/business-central.adoc [hunk 2]
```diff
@@ -120,20 +124,20 @@ In order to turn this decision table into non-guided form so that it can be modi
...
 image::../images/bc-example-guided-dt-export.png[Exporting a Guided Decision Table in Business Central]
 
-Once the table has been converted to XLS format, the new file is added to the Business Central `Project Explorer` view:
+After the table is converted to XLS format, the new file is added to the Business Central `Project Explorer` view.
 
 .Converted Guided Decision Table in Business Central
 image::../images/bc-example-guided-dt-export-navigator.png[Converting a Guided Decision Table in Business Central]
 
-From there it is simply a matter of selecting the converted file and pressing one of the `Download` buttons in Business Central, which will download the converted spreadsheet file to your local machine so the file can be added to your project's `src/main/resources` folder of your Maven project.
+Select the converted file and choose one of the Download buttons in Business Central, which download the converted spreadsheet file to your local machine. You can add this file to src/main/resources folder of your Maven project.
```
Comment (LightGuard) [inline migration-guide/business-central.adoc:140]: Line should break before "You can"

**Review by LightGuard — CHANGES_REQUESTED** [review]
Just one small thing from me.

**Review by LightGuard — APPROVED** [review]

**Pull request merged**